gitpod-io/gitpod
# Upgrade platform version of JetBrains Backend Plugin (EAP)

## Description


## Related Issue(s)

Fixes ENT-822

## How to test

~~- Check integration tests of current PR's build, most of them should be passed~~ integration test failed because Gateway Plugin is not updated

We triggered dev JetBrains Nightly Job to update dev EAPs, so you may need to run leeway script to update preview view env's JetBrains IDE images
```
cd /workspace/gitpod/components/ide/gha-update-image
leeway run .:jb-use-dev-latest
```

Test manually with preview env:
- Set use latest editors in user settings
- Open workspace with `IDEA`, `Goland`, `RubyMine`, `WebStorm`, `PyCharm`, `PhpStorm` (other editors are broken), they should work well. example repo: https://github.com/gitpod-samples/spring-petclinic

|IDEA + GoLand + RubyMine|WebStorm + PyCharm + PhpStorm|
|:-:|:-:|
<img width="1667" alt="image" src="https://github.com/user-attachments/assets/0b256396-e53f-439b-aea6-ce124554747b">|<img width="1529" alt="image" src="https://github.com/user-attachments/assets/f32dfbd8-17a7-4170-aef1-f90e4b38c816">|

## Documentation


#### Preview status

<p>Gitpod was successfully deployed to your preview environment.</p>
<ul>
	<li><b>🏷️ Name</b> - hw-jb-eap</li>
	<li><b>🔗 URL</b> - <a href="https://hw-jb-eap.preview.gitpod-dev.com/workspaces" target="_blank">hw-jb-eap.preview.gitpod-dev.com/workspaces</a>.</li>
	<li><b>📚 Documentation</b> - See our <a href="https://www.notion.so/gitpod/6debd359591b43688b52f76329d04010#7c1ce80ab31a41e29eff2735e38eec39" target="_blank">internal documentation</a> for information on how to interact with your preview environment.</li>
	<li><b>📦 Version</b> - hw-jb-eap-gha.29061</li>
	<li><b>🗒️ Logs</b> - <a href="https://console.cloud.google.com/logs/query;query=jsonPayload.kubernetes.host%3D%22preview-hw-jb-eap%22%0A%0A--%20Filter%20on%20service:%0A--%20jsonPayload.serviceContext.service%3D%22ws-manager-mk2%22%0A;duration=P1D?project=gitpod-dev-preview" target="_blank">GCP Logs Explorer</a></li>
</ul>

## Build Options

/hold
